ROADMAP.md

```diff
@@ -4,32 +4,37 @@ This document defines a high level roadmap for Karmada development and upcoming
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep karmada progressing and shipping exciting and valuable features.
 
+## 2022 H1
+- Multi-cluster HA scheduling policy
+  * spread by region
+  * spread by zone
+  * spread by provider
+- Multi-cluster Ingress
+- Multi-cluster HPA (Horizontal Pod Autoscaling)
+- Federated resource quota
+- API reference
+- [Karmada website](https://karmada.io/) refactor
+- Policy-based governance, risk, and compliance
+- Multi-cluster DNS (cluster identity)
+- Global search across clusters
+- Scheduling re-balancing
 
-## 2021 Q1
-- K8s native API support for multi-cluster applications
-- Multi-cluster application status aggregation
-- Per-cluster configuration customization for applications
-- Cluster Affinity Scheduling Policy
-- Multi-cluster HA scheduling policy: spread by cluster
-- Cluster registry; Support register existing clusters
-
-## 2021 Q2
-- Multi-cluster application automatic migration (DR)
-- Dynamically split replicas of applications based on multiple dimensions
-- Taint Toleration for clusters
-- Multiple sync mode: Push, Pull
+## 2022 H2
+- Karmada Dashboard - alpha release
+- Karmada scalability baseline (performance report)
+- Cluster addons
+- Helm chart propagation
+- Multi-cluster events
+- Multi-cluster Operator specifications
+- Multi-cluster Application
+- Multi-cluster monitoring
+- Multi-cluster logging
+- Multi-cluster storage
+- Multi-cluster RBAC
```

Comment:
Is there a plan to support clusterset and clusterset rbac? refer to https://github.com/clusternet/clusternet @RainbowMango

Reply:
Good question. 
If what you are asking is the `clusterset`  defined by [multi-cluster-service-api](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#terminology), the answer is yes.

Reply:
Not exactly,`clusterset `defined by `multi-cluster-service-api` is a placeholder name for a group of clusters with a high degree of mutual trust and shared ownership that share services amongst themselves. What i mean clusterset is a clusterd scoped API  in the control plane for grouping a few managed clusters into a “set”, so that control plane admin can operate these clusters altogether in a higher level.

Reply:
> What i mean clusterset is a clusterd scoped API in the control plane for grouping a few managed clusters into a “set”, so that control plane admin can operate these clusters altogether in a higher level.

They are the same thing I think.  Is there any use case?

Reply:
> They are the same thing I think

ok 



> Is there any use case

yes,we are plan to integration karmada in our system, cluster group and cluster group rbac is necessary for us

Reply:
Great! I think we can have a talk at next community meeting. If you need it, we can consider to do it at next release.

Reply:
We meet first problem is namespace propagate to all cluster. In our system we treat namespace as project assign to a group of users in some clusters. If namespace sync propagate all cluster, other groups of user who owned 
another part of the cluster will see this namespace and can not delete it. Currently we resolve this problem by disable `namespac-econtroller`.